assets/js/modules/analytics-4/components/widgets/PopularProductsWidget.js

```diff
@@ -144,15 +161,17 @@ export default function PopularProductsWidget( props ) {
 		return <WidgetNull />;
 	}
 
-	const infoTooltip = createInterpolateElement(
-		__(
-			'Site Kit detected these are your product pages. If this is inaccurate, you can <a>replace</a> this with another metric',
-			'google-site-kit'
-		),
-		{
-			a: <Link onClick={ openMetricsSelectionPanel } />,
-		}
-	);
+	const infoTooltip = showTooltip
```

Comment:
This block doesn't need to change as we can inline the condition when passing the prop instead.